homeassistant/components/dsmr_reader/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "config": {
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "mqtt_missing": "The MQTT service is required for this integration."
+    },
+    "step": {
+      "confirm": {
+        "description": "Make sure to configure the 'split topic' data sources in DSMR Reader, as described [in the documentation](https://www.home-assistant.io/integrations/dsmr_reader#setup)."
```

Comment:
please insert the link via a `description_placeholder`

Reply:
Done. I didn't do it at first because then I have to also override the `async_step_confirm` method, but I guess it's not that much of an effort anyway.